GitUI/UserControls/FileStatusList.cs

```diff
@@ -1518,6 +1538,12 @@ public override int Compare(ListViewItem x, ListViewItem y)
 
         #endregion
 
+        #region private Color constants
+        //// Do not declare the colors "static" because Color.FromArgb() will not work at their initialization.
+        private readonly Color _activeInputColor = Color.FromArgb(0xC8, 0xFF, 0xC8);
```

Comment:
the active color is too visible for my preference but let someone better at UI clean this up...

Reply:
It is the same color as used for new lines in the diff view as agreed upon in the feature request.